backend/src/main/java/com/bakdata/conquery/apiv1/query/TableExportQuery.java

```diff
@@ -195,27 +203,31 @@ private Map<SecondaryIdDescription, Integer> calculateSecondaryIdPositions(Atomi
 		return secondaryIdPositions;
 	}
 
-	private static List<ResultInfo> createResultInfos(Map<SecondaryIdDescription, Integer> secondaryIdPositions, Map<Column, Integer> positions, @NotEmpty @Valid List<CQConcept> tables, boolean rawConceptColumns) {
+	private List<ResultInfo> createResultInfos(Map<SecondaryIdDescription, Integer> secondaryIdPositions) {
 
 		final int size = positions.values().stream().mapToInt(i -> i).max().getAsInt() + 1;
 
 		final ResultInfo[] infos = new ResultInfo[size];
 
-		infos[0] = ConqueryConstants.DATES_INFO;
+		final Set<SecondaryIdDescription> grouping = previewConfig.getGrouping();
```

Comment:
Die PreviewConfig Anpassungen würde ich in die EntityPreviewExecution#getResultInfos() packen.

Reply:
Ich weiß  nicht genau was du meinst mit Anpassungen aber so ganz unkompliziert ist das nicht, dass da hin zu verfrachten

Reply:
Ich würde gerne dass der TableExport nichts von der PreviewConfig wissen muss.

Es wäre kompliziert, weil du ein SimpleResultInfo nutzt für die SecondaryIds und auf Ebene von EntityPreviewExecution#getResultInfos() nicht mehr sehen kannst welches resultInfo zu einer SecondaryId gehörte um dann Group (über die semantics könnte das noch gehen, die sind ja mit SecondaryID geflagged, aber dann hast du auch nur das Label).

Bei Hidden sollte das aber kein Problem sein da. Da hast du den ColumnResultInfo eingeführt und

Reply:
> Ich würde gerne dass der TableExport nichts von der PreviewConfig wissen muss.

Der TableExport wurde aber (ursprünglich) extra dafür entwickelt. Aber verstehe deinen Punkt, es ist glaube ich eine relativ große Geschichte.

Wäre es ok für dich, wenn ich das as-is merge und nach meinem Urlaub anpasse?

Reply:
Ja das passt so :)